src/ngraph/frontend/onnx_import/core/null_node.cpp

```diff
@@ -0,0 +1,36 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "null_node.hpp"
+#include "ngraph/node.hpp"
+#include "ngraph/node_vector.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        NullNode::NullNode()
+            : ngraph::Node("NullNode", {}, 0)
+        {
+            m_name = "NullNode";
```

Comment:
Overriding the Node's m_name in this way is a bad idea. Why is this done? A comment why would be nice if this is really necessary.

Reply:
Good point I will remove it.